### DISCUSSION TITLE
How to forward from cloudwatch Logs to security Onion v2.3

### DISCUSSION THREAD

**Discussion by ITgeevoSOC** [Security-Onion-Solutions/securityonion#5682] [Unsupported Versions] [answered]
Hi,

Please can you inform us if it's possible to forward from cloudwatch Logs to security Onion v2.3.
If yes, how this be done.

Regards

**Comment by weslambert** [comment, marked as answer]
You can use the AWS Filebeat module with the  Cloudwatch  configuration to achieve this:

https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-aws.html#_module_configuration

Similarly, we have a walkthrough for Cloudtrail log ingestion here: 
https://docs.securityonion.net/en/latest/filebeat.html#walktrough-aws-cloudtrail-logs

**Reply by ITgeevoSOC** [reply]
Thank you for your help.
